bigframes/pandas/__init__.py

```diff
@@ -695,6 +695,48 @@ def to_datetime(
 to_datetime.__doc__ = vendored_pandas_datetimes.to_datetime.__doc__
 
 
+def get_default_session_id() -> str:
+    """Gets the default session id. This is the session id that is
+    used whenever a custom session has not been provided. It is the
+    session id of the default global session. It is prefixed to the
+    table id of all temporary tables created in the global session.
+
+    Returns:
+        str, the default global session id, ex. 'sessiona1b2c'
+    """
+    return get_global_session().session_id
+
+
+def clean_up_by_session_id(session_id: str) -> None:
```

Comment:
Let's require project and location (or maybe make them optional parameters and use the `_anonymous_dataset` on the global session if one of those isn't passed in).

Reply:
Roger, added the suggested optional behavior and checked that neither or both are passed (since we don't want to silently ignore it if someone just passes one.)